plugins/apple/swift/v1.27.0/extramoduleimports.patch

```diff
@@ -0,0 +1,60 @@
+diff --git a/Sources/protoc-gen-swift/FileGenerator.swift b/Sources/protoc-gen-swift/FileGenerator.swift
+index ebbbdca6..0e21fa95 100644
+--- a/Sources/protoc-gen-swift/FileGenerator.swift
++++ b/Sources/protoc-gen-swift/FileGenerator.swift
+@@ -113,6 +113,14 @@ class FileGenerator {
+             p.print(neededImports)
+         }
+ 
++        let neededCustomImports = generatorOptions.extraModuleImports
++        if !neededCustomImports.isEmpty {
++            p.print()
++            for i in neededCustomImports {
++                p.print("import \(i)\n")
++            }
++        }
++
+         p.print()
+         generateVersionCheck(printer: &p)
+ 
+diff --git a/Sources/protoc-gen-swift/GeneratorOptions.swift b/Sources/protoc-gen-swift/GeneratorOptions.swift
+index 2635cb8d..58a75681 100644
+--- a/Sources/protoc-gen-swift/GeneratorOptions.swift
++++ b/Sources/protoc-gen-swift/GeneratorOptions.swift
+@@ -49,6 +49,7 @@ class GeneratorOptions {
+     }
+   }
+ 
++  let extraModuleImports: [String]
```

Comment:
Moved some of the constants and variables towards the "top" to minimize conflicts.